### ISSUE TITLE
List position and total items are not announced in message list

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2780]
**Describe the bug**

Previously, specifically in Unigram 8.8 (7407) X64 Direct, the messages list was announced just like the chat list. Now, maybe having to do with the messages being checkboxes (but regardless I think they shouldn't be mutually exclusive) the position of items and the total is not announced (for example, 1 of 100, if I'm at the top of a chat of 100 messages).

**To Reproduce**
1. Open narrator.
2. Open any chat.
3. Arrow up and down.
4. Now, open the chat list and do the same.
5. Notice that in the chat list the position and total is announced.

**Expected behavior**

The list position and item count should be announced.

**Version Info**
 - Unigram: Unigram 8.8 (7436) X64 Direct
 - Windows: Versión 21H2 (compilación de SO 19044.1766)

**Comment by FrayxRulez** [comment]
Is this information really necessary considering that the count and position previously announced was not the real one?

**Comment by sukiletxe** [comment]
Wasn't it? I believe that, except when scrolling, it was.

**Comment by FrayxRulez** [comment]
@sukiletxe It was, but just relatively to the currently loaded messages. What's the point of knowning that there's currently 50 messages in the list when you might have millions in that chat?

**Comment by sukiletxe** [comment]
If they are the first or the last ones, knowing how many are left to read.

**Comment by FrayxRulez** [comment]
This has been implemented at some point
